xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/common/LedgerIndex.java

```diff
@@ -85,6 +109,24 @@ public LedgerIndex plus(LedgerIndex other) {
     return plus(other.unsignedLongValue());
   }
 
+  /**
+   * Check whether or not the index wrapped by this {@link LedgerIndex} is valid.
+   * A valid index is a shortcut or an unsigned long.
+   *
+   * @return true if the wrapped index is valid, false otherwise.
+   */
+  public boolean isValid() {
+    if (value.equals("current")) return true;
+    if (value.equals("validated")) return true;
+    if (value.equals("closed")) return true;
+    try {
+      unsignedLongValue();
+      return true;
+    } catch (NumberFormatException e) {
+      return false;
```

Comment:
Is this a phantom comment, or is this actually not covered? Seems like line 31 of the unit test should trigger this. Am I missing something? Might be worth "run with coverage" in Intellij just to double-check.

Reply:
I was confused about this as well. I can pull this down and run with coverage. Sometimes Codecov isn't 100% accurate

Reply:
Yes this should have been considered covered, but jacoco seems to not report `throw` statements as covered.
From [the jacoco FAQ](https://www.eclemma.org/jacoco/trunk/doc/faq.html):

**Source code lines with exceptions show no coverage. Why?**

*JaCoCo determines code execution with so called probes. Probes are inserted into the control flow at certain positions. Code is considered as executed when a subsequent probe has been executed. In case of exceptions such a sequence of instructions is aborted somewhere in the middle and the corresponding line of source code is not marked as covered.*

I fixed this in my latest commit, now the code coverage should be OK.